### ISSUE TITLE
Tested games

### ISSUE THREAD

**Issue by anticitizn** [anticitizn/creamlinux#3]
If you have tried creamlinux with a game beyond those listed in the readme, please leave a quick message here.

**Comment by joakimgaming** [comment]
Works with eu4 v1.34.5.0 and creamlinux v2.0.1

**Comment by anticitizn** [comment]
Thank you, added to tested games

**Comment by akeryuuth** [comment]
Seems to work with Stellaris 3.7.4 (DLC files are 3.7.2 I think?) and creamlilnux 2.0.2

**Comment by anticitizn** [comment]
Hey @rednatnas, I noticed you tried out creamlinux with Civilization VI. Any luck?

**Comment by rednatnas** [comment]
Sadly, Civilization VI doesn't work. I also read that it's better to use the Windows version with Proton, the native Linux version has quite a few bugs. Still, Windows CreamApi + Proton does work.

**Comment by LDprg** [comment]
> Sadly, Civilization VI doesn't work. I also read that it's better to use the Windows version with Proton, the native Linux version has quite a few bugs. Still, Windows CreamApi + Proton does work.

Yeah I would also recommend using the proton version off the game. Mainly because of multiplayer since you can't play with friends using windows if you don't use proton. For me the native version was sometimes very buggy and laggy compared to proton. Creamapi works fine with Civ VI  using proton.

**Comment by Ruberine** [comment]
> Seems to work with Stellaris 3.7.4 (DLC files are 3.7.2 I think?) and creamlilnux 2.0.2

@aurelienLavanchy Are they still working? Attempting to use this now, I'm seeing "Unable to verify ownership of DLC" on all of them in the launcher. None appear in-game.

**Comment by Vaalvist** [comment]
Victoria 3 works for me

**Comment by anticitizn** [comment]
Thanks, added to the list of tested games
